src/sass/login.scss

```diff
@@ -1,5 +1,186 @@
 @import "shared-variables";
 
+.login {
+  background: $color-white;
+
+  hr {
+    border-color: $color-gray-lighter;
+  }
+
+  @include media-maxwidth($small-screen) {
+    .hide-for-small {
+      display: none;
+    }
+
+    .help-info {
+      padding: 2em !important;
```

Comment:
@bshyong one more...this padding still needs to go. Then I _think_ we should be good!

![image](https://user-images.githubusercontent.com/4106356/30384945-817216c4-985a-11e7-8bc5-85b16370229f.png)